packages/wdio-webdriver-mock-service/src/index.js

```diff
@@ -55,6 +56,15 @@ export default class WebdriverMockService {
         this.command.elementClick(ELEMENT_ID).once().reply(200, { value: null })
     }
 
+    isExistingScenario() {
+        this.nockReset()
+        const elemResponse = { 'element-6066-11e4-a52e-4f735466cecf': ELEMENT_ID }
+
+        this.command.findElement().times(1).reply(200, { value: elemResponse })
+        this.command.findElementFromElement(ELEMENT_ID).times(2).reply(200, { value: elemResponse })
+        this.command.findElementsFromElement(ELEMENT_ID).times(2).reply(200, { value: [elemResponse] })
```

Comment:
this is not needed
```suggestion
```

Reply:
can remove this one because calling isExisting() calls findElementsFromElement underneath

Reply:
right